preview/arcus-general.md

```diff
@@ -0,0 +1,71 @@
+# Arcus - Azure development in a breeze.
+
+## What is Arcus?
+A set of Codit-developed OSS libraries/tools that simplify building apps running on Microsoft Azure.
+It is Codit’s strategy to centralize company-wide best practices and use it across all our customers.
+
+* Built By Experts, For Experts
+  Sponsored by Codit! Maintained by a dedicated set of Codit experts that assures quality for our current and future clients.
+
+* Modular Design
+  All Arcus components are highly functional and composable. They can be used separately, but they are better together.
+
+* Open-Source & Free to use
+  The Arcus framework is constantly updated by contributions from Codit employees and abroad!
```

Comment:
It is true that most (if not all) contributions are made by Codit employees, but I'd say we're open for contributions by anyone.

Reply:
That's the `...and abroad` part, no? Or would you be more explicit about that?

Reply:
Yes, I would be more explicit.
Something like:

```suggestion
  The Arcus library is constantly updated by contributions from Codit employees and abroad!  
  As the Arcus library is free to use and open-source so we happily accept your contributions!  Please take a look at our [contribution guide](https://github.com/arcus-azure/.github/blob/master/CONTRIBUTING.md) to get up to speed if you want to contribute.
```
(Are we talking about a framework or a library; imho, Arcus is a library and not a framework).

Reply:
Librarie(s), 😄 , but yeah, is good for me.